### PR TITLE
Create InputField component

### PR DESCRIPTION
**Pull request by KGhising**
add component for input field.

### CODE REVIEW

**Comment by silwalanish** [comment]
@KGhising Please add in the description for the PR. Explain what you did.
Also, make sure the checks are passing. The lint checks are failing. 
Run `yarn lint` in local environment to find the issue.

File: src/Components/InputField.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import '../Themes/InputField.css';
+
+const InputField = (props) => {
+  return (
+    <input placeholder={props.placeholder} type={props.type} />
+  )
+}
+export default InputField;
```
Comment (silwalanish) [inline src/Components/InputField.js:10]: Add a new line at the end of the file.

File: src/Themes/InputField.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+input{
+  width: 300px;
+  height: 30px;
+  border: 2px solid yellow;
+  border-radius: 9px;
+  padding-left: 10px;
+}
```
Comment (silwalanish) [inline src/Themes/InputField.css:7]: Add a new line at the end of the line

File: src/Components/InputField.js (hunk 1 above)
Comment (KGhising) [reply src/Components/InputField.js:10]: Okay

File: src/Components/InputField.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import '../Themes/InputField.css';
+
+export default class InputField extends React.Component{
+  render(){
+    let placeholder = 'Full name';
```
Comment (silwalanish) [inline src/Components/InputField.js:7]: Use props here. If you are getting lint errors. Pull the latest master branch and merge it to your branch

File: src/Components/InputField.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import '../Themes/InputField.css';
+
+export default class InputField extends React.Component{
+  render(){
+    return (
+      <input placeholder={this.props.placeholder} type={this.props.type} />
```
Comment (silwalanish) [inline src/Components/InputField.js:8]: Let's add another props for `defaultValue`

File: src/Themes/InputField.css [hunk 5]
```diff
@@ -0,0 +1,7 @@
+input{
```
Comment (silwalanish) [inline src/Themes/InputField.css:1]: let's apply style to a class instead of element itself.

File: src/Components/InputField.js (hunk 4 above)
Comment (silwalanish) [inline src/Components/InputField.js:8]: Also, let's add icon, help text and error text. We need to be able to do all this with this component.
image

Comment (silwalanish) [reply src/Components/InputField.js:8]: @KGhising Maybe make a component called `Icon` that uses either `material-icons` or `font-awesome` and takes a `name` as prop for the icon name. That can be used anywhere.

Comment (KGhising) [reply src/Components/InputField.js:8]: okay

File: src/Themes/InputField.css [hunk 6]
```diff
@@ -0,0 +1,13 @@
+.inputfield{
```
Comment (silwalanish) [inline src/Themes/InputField.css:1]: Also add a focus state for inputs.
```suggestion
.input-field{
```

File: src/Components/Icon.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {faUser, faEnvelope, faKey, faEye, faEyeSlash} from '@fortawesome/free-solid-svg-icons';
+
+library.add(faUser, faEnvelope, faKey, faEye, faEyeSlash);
```
Comment (silwalanish) [inline src/Components/Icon.js:7]: Add all icons. We don't want to come and modify this all the time.

File: src/index.js [hunk 8]
```diff
@@ -2,5 +2,6 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 
 import App from './App.js';
+import './Components/Icon';
```
Comment (silwalanish) [inline src/index.js:5]: Why is this required?

**Comment by silwalanish** [comment]
Closing this. As #4 does the same.
